CoreAardvark.podspec

```diff
@@ -1,12 +1,13 @@
 Pod::Spec.new do |s|
   s.name     = 'CoreAardvark'
-  s.version  = '2.0.0'
+  s.version  = '2.1.0'
   s.license  = 'Apache License, Version 2.0'
   s.summary  = 'Aardvark is a library that makes it dead simple to create actionable bug reports. Usable by extensions.'
   s.homepage = 'https://github.com/square/Aardvark'
   s.authors  = 'Square'
   s.source   = { :git => 'https://github.com/square/Aardvark.git', :tag => "CoreAardvark/#{ s.version.to_s }" }
   s.ios.deployment_target = '8.0'
+  s.watchos.deployment_target = '3.0'
```

Comment:
Build is failing. Looks like watch doesn't know about `UIApplicationWillTerminateNotification`.